pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -980,6 +980,9 @@ func validateBootOrder(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec
 
 	// Validate that volumes match disks and filesystems correctly
 	for idx, volume := range spec.Volumes {
```

Comment:
Seems pretty straightforward, but maybe just small test in `pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go` can close the loop? not insisting

Reply:
I am aware that there a some UT that can be added, I want this code to get inside before feature freeze and I'll add them in the stabilization phase

Reply:
@akalenyu I was able to work on all the needed UT and I added them either to the commit they were missing or to commit of "add UTs"